EduVPN-multiOS/Coordinators/TunnelProviderManagerCoordinator.swift

```diff
@@ -125,6 +125,7 @@ class TunnelProviderManagerCoordinator: Coordinator {
                 self.configureVPN({ _ in
                     var builder = OpenVPNTunnelProvider.ConfigurationBuilder(sessionConfiguration: configBuilder.build())
                     builder.masksPrivateData = false
+                    builder.shouldDebug = true
```

Comment:
I am a bit wary of this. Debugging is for developers, not for users to see their connection log. Also users may be asked to share their log, in which case perhaps private data should be masked?

Reply:
The name `shouldDebug` might suggest it logs more info, but if we look at the TunnelKit code, it only controls whether the log is persisted ([urlForLog], [flushLog]), and whether the log is printed to the console ([startTunnel], [configureLogging]).

[urlForLog]: https://github.com/passepartoutvpn/tunnelkit/blob/v2.2.4/TunnelKit/Sources/Protocols/OpenVPN/AppExtension/OpenVPNTunnelProvider%2BConfiguration.swift#L249
[flushLog]: https://github.com/passepartoutvpn/tunnelkit/blob/v2.2.4/TunnelKit/Sources/Protocols/OpenVPN/AppExtension/OpenVPNTunnelProvider.swift#L817
[startTunnel]: https://github.com/passepartoutvpn/tunnelkit/blob/v2.2.4/TunnelKit/Sources/Protocols/OpenVPN/AppExtension/OpenVPNTunnelProvider.swift#L206
[configureLogging]: https://github.com/passepartoutvpn/tunnelkit/blob/v2.2.4/TunnelKit/Sources/Protocols/OpenVPN/AppExtension/OpenVPNTunnelProvider.swift#L800